docs/nodes/nodes-library/nodes/Wekan/README.md

```diff
@@ -0,0 +1,164 @@
+---
+permalink: /nodes/n8n-nodes-base.wekan
+description: Learn how to use the Wekan node in n8n
+---
+
+# Wekan
+
+[Wekan](https://wekan.github.io/) is an open-source kanban board that allows a card-based task and to-do management.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Wekan/README.md).
+:::
+
+## Basic Operations
+
+::: details Board
+- Create a new board
+- Delete a board
+- Get the data of a board
+- Get all user boards
+:::
+
+::: details Card
+- Create a new card
+- Delete a card
+- Get a card
+- Update a card
+:::
+
+::: details Card Comment
+- Create a comment on a card
+- Delete a comment from a card
+- Get a card comment
+- Get all card comments
+:::
+
+::: details Checklist
+- Create a new checklist
+- Delete a checklist
+- Get the data of a checklist
+- Returns all checklists for the card
+:::
+
+::: details Checklist Item
+- Delete a checklist item
+- Get a checklist item
+- Update a checklist item
+:::
+
+::: details List
+- Create a new list
+- Delete a list
+- Get the data of a list
+- Get all board lists
+:::
+
+## Example Usage
+
+This workflow allows you to create a board, lists, and a card using the Wekan node. The workflow also allows you to update the card using the Wekan node. You can also find the [workflow](https://n8n.io/workflows/728) on n8n.io. This example usage workflow uses the following nodes.
```

Comment:
It seems much more is happening in the example workflow as opposed to what is explained here. Could you please update? Feel free to split that into multiple statements.